popup.js

```diff
@@ -32,7 +32,7 @@ function refineResults(searchResults, query) {
     const queryLen = query.length;
     const queryLower = query.toLowerCase();
```

Comment:
both variable definition lines are unnecessarily verbose since you're only using these once. Just call them on query below